yaksh/models.py

```diff
@@ -401,6 +402,12 @@ def set_extract_status(self):
             self.extract = True
         self.save()
 
+    def set_hide_status(self):
```

Comment:
Change this to `toggle_hide_status` or you can just pass this as an argument and set the value and save it.